nservicebus/upgrades/5to6.md

```diff
@@ -632,7 +633,7 @@ snippet: 5to6-MsmqSubscriptionAuthorizer
 
 ## Synchronous request-response (callbacks)
 
-The synchronous request-response feature, also known as callbacks, has been moved from the NServiceBus core to the separate Nuget package [NServiceBus.Callbacks](https://www.nuget.org/packages/NServiceBus.Callbacks/). That package must be used in order to use the callback functionality in Version 6. 
+The synchronous request-response feature, also known as callbacks, has been moved from the NServiceBus core to the separate Nuget package [NServiceBus.Callbacks](https://www.nuget.org/packages/NServiceBus.Callbacks/). That package must be used in order to use the callback functionality in Version 6.
```

Comment:
i dont mind them being clean up. i do it when i am doing a full review of the pages. just saying that if u do a PR and u touch a line it looks like you have edited it. similar to if in a cs file u change one line then submit the PR with a full code cleanup.

So perhaps do a cleanup + direct commit before doing a PR.